--- lib/os_io.c~	2019-01-18 20:10:55.000000000 +0100
+++ lib/os_io.c	2020-07-29 20:29:51.525331737 +0200
@@ -1561,6 +1561,14 @@
   self->close_direction = self->direction;
 }

+void ___os_device_transfer_close_responsibility
+   ___P((___device *self),
+        (self)
+___device *self;)
+{
+  device_transfer_close_responsibility(self);
+}
+
 ___HIDDEN void device_add_ref
    ___P((___device *self),
         (self)
